packages/asc-core/package.json

```diff
@@ -26,8 +26,8 @@
     "test": "echo \"Error: run tests from root\" && exit 1"
   },
   "sideEffects": false,
-  "dependencies": {
-    "styled-components": "^4.3.2"
+  "peerDependencies": {
+    "styled-components": ">= 4.4.1"
```

Comment:
This version range also includes `styled-component` v5, do we want that?

Reply:
Yes, as v5 doesn't have breaking changes: https://github.com/styled-components/styled-components/releases

Reply:
:ok_hand: :ok: